thrift/src/test/java/com/linecorp/armeria/it/client/retry/RetryingRpcClientTest.java

```diff
@@ -60,7 +77,7 @@ protected void configure(ServerBuilder sb) throws Exception {
     public void execute() throws Exception {
         HelloService.Iface client = new ClientBuilder(server.uri(BINARY, "/thrift"))
                 .decorator(RpcRequest.class, RpcResponse.class,
-                           RetryingRpcClient.newDecorator(ALWAYS_HANDLES_EXCEPTION))
+                           new RetryingRpcClientBuilder(ONLY_HANDLES_EXCEPTION).newDecorator())
```

Comment:
Could we add (back) the `newDecorator()` methods to `RetryingRpc/HttpClient` for the most common cases?